KsApi/models/Project.swift

```diff
@@ -6,6 +6,7 @@ import Prelude
 public struct Project {
 
   public private(set) var blurb: String
+  public private(set) var staffPick: Bool
```

Comment:
A smaller thing here... We try to keep the properties of the struct alphabetized (just like we do with the list of inputs/outputs functions).